actions/ocr.go

```diff
@@ -30,7 +30,7 @@ func DeployOCRContracts(
 			linkTokenContract.Address(),
 			contracts.DefaultOffChainAggregatorOptions(),
 		)
-		Expect(err).ShouldNot(HaveOccurred())
+		Expect(err).ShouldNot(HaveOccurred(), "Deploying OCR instance %d shouldn't fail", i+1)
```

Comment:
I know originally we were going to ignore putting annotation on `Expect(err).ShouldNot(HaveOccurred())` checks since they provide a pretty clear message when they occur. I am not saying I have a problem with adding these, just stating what I last heard on the subject.